manual/CHAPTER_CellLib.tex

```diff
@@ -273,6 +274,29 @@ \subsection{Memories}
 The cell with the higher integer value in this parameter wins a write conflict.
 \end{itemize}
 
+The {\tt \$meminit} cells have an address input \B{ADDR} and a data input \B{DATA}, with the width
+of the \B{DATA} port equal to \B{WIDTH} parameter times \B{WORDS} parameter. Both of the inputs
+must resolve to a constant for synthesis to succeed. If two {\tt \$meminit} cells have the same
+\B{MEMID} parameter and \B{ADDR} input, the contents of that memory at that address is undefined;
+the \B{PRIORITY} parameter is currently ignored by the {\tt memory\_collect} pass.
```

Comment:
Redundant comment to force github into "changes requested" mode: `memory_collect` _does_ use PRIORITY to resolve write conflicts.

Reply:
Ah, I misread--fixed now!